test/validate/eia_test.py

```diff
@@ -45,15 +45,15 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_521_304, 1_521_304, 127_412),
+        ("bf_eia923", 1_559_257, 1_559_257, 133_754),
         ("bga_eia860", 141_638, 141_638, 141_638),
-        ("boil_eia860", 80_059, 80_059, 80_059),
-        ("frc_eia923", 631_975, 258_115, 25_370),
+        ("boil_eia860", 83_354, 83_354, 83_354),
+        ("frc_eia923", 639_647, 261_583, 25_370),
         ("gen_eia923", None, 5_178_892, 433_286),
-        ("gens_eia860", 556_578, 556_578, 556_578),
-        ("gf_eia923", 2_859_269, 2_859_269, 244_796),
+        ("gens_eia860", 556_586, 556_586, 556_586),
+        ("gf_eia923", 2_879_896, 2_879_896, 244_796),
```

Comment:
do you have any sense of why the annual rows didn't increase for some of these?

Reply:
I don't. But I was also thinking about this. When I dig into the issues with the `gen_eia923` table I'll look at this too

Reply:
This is now a non-issue because we remove the ytd records from the annual data anyways.